src/go/rdctl/cmd/set.go

```diff
@@ -58,9 +60,10 @@ var setCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(setCmd)
 	setCmd.Flags().StringVar(&specifiedSettings.ContainerEngine, "container-engine", "", "Set engine to containerd or moby (aka docker).")
-	setCmd.Flags().BoolVar(&specifiedSettings.Enabled, "kubernetes-enabled", false, "Control whether kubernetes runs in the backend.")
+	setCmd.Flags().BoolVar(&specifiedSettings.Enabled, "kubernetes-enabled", true, "Control whether kubernetes runs in the backend.")
```

Comment:
Because I read the documentation and saw that if they specify `--kubernetes-enabled` we want it to be `true`. Or did I get something else wrong?

Reply:
This is the default value (as in, if the flag was never explicitly specified by the user), right? Wouldn't that default to false? Otherwise we'd always want `--kubernetes-enabled=false` (as opposed to just not specifying it) if we don't want Kubernetes?

Reply:
Mark is correct -- it's the default value for the flag. But for the `set` and (soon to be PRed `start` command), we only process flags that are actually specified by consulting the `Changed` property. From the docs at https://pkg.go.dev/github.com/spf13/pflag#FlagSet.Changed:

```
Changed returns true if the flag was explicitly set during Parse() and false otherwise 
```

So better to be consistent, although those default values don't matter (so far).